mlog/human/default_writer.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package human
+
+import (
+	"fmt"
+	"io"
+	"strings"
+	"time"
+)
+
+type SimpleWriter struct {
+	out io.Writer
+}
+
+func (w *SimpleWriter) Write(e LogEntry) {
+	fmt.Fprintln(w.out, e)
+}
+
+func NewSimpleWriter(out io.Writer) *SimpleWriter {
+	w := new(SimpleWriter)
+	w.out = out
+	return w
+}
+
+func (f LogEntry) String() string {
```

Comment:
This function is located in this file, rather than where the struct is defined, as it pertains to what the simple writer will output?

Reply:
I think this should go with the LogEntry struct and not here.

Reply:
Yeah that was my logic for putting it here. I think the logic belongs in this file, but it does perhaps make sense to not make it a regular function instead of a struct method.

Reply:
Moved to a new file